include/wlr/render/pixman.h

```diff
@@ -9,9 +9,30 @@
 #ifndef WLR_RENDER_PIXMAN_H
 #define WLR_RENDER_PIXMAN_H
 
+#include <pixman.h>
 #include <wlr/backend.h>
 #include <wlr/render/wlr_renderer.h>
 
 struct wlr_renderer *wlr_pixman_renderer_create(void);
 
+struct wlr_pixman_texture_attribs {
+	void *data;
```

Comment:
What is this `data` pointer used for?

Reply:
Not use in caller. This is copy from struct wlr_pixman_texture.

Reply:
We add `data` pointers in structures allocated by wlroots to allow compositors to attach their own state to wlroots' objects. However this struct is not allocated by wlroots, so I'd rather not add this field.

Reply:
Ok, I  haved remove it form wlr_pixman_texture_attribs, it is only contains pixman_image_t now.